cobalt/build/configs/linux-x64x11/args.gn

```diff
@@ -46,3 +46,11 @@ enable_nacl = false
 enable_print_preview = false
 use_cups = false
 use_kerberos = false
+
+include_transport_security_state_preload_list = false
+build_epoll_based_tools = false
+enable_unix_sockets = false
+use_remote_test_server = false
```

Comment:
We shouldn't need to override this one explicitly, as `enable_python_utils = true` defaults to true on posix/linux